pkg/storage/sqlcommon/encoding.go

```diff
@@ -0,0 +1,21 @@
+package sqlcommon
+
+import (
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+func marshalRelationshipCondition(
+	rel *openfgav1.RelationshipCondition,
+) (string, *[]byte, error) {
```

Comment:
Can we return `[]byte` here instead? `var marshaledBytes []byte = nil` can be treated the same as `var marshaledBytes *[]byte = nil`. The extra pointer appears to be redundant.

Reply:
ah yep good catch forgot to change this one here too